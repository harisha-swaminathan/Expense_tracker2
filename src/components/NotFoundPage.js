import {Link } from 'react-router-dom';

import React from 'react';

const NotFoundPage = () => {
    return (
        <div>
            404! Page not found. <Link to="/"> Go home!</Link>
        </div>
    )
};

export default NotFoundPage;