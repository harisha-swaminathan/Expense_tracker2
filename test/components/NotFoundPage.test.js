import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';

import NotFoundPage from '../../src/components/NotFoundPage';

test('component is rendered as expected', () =>{
    const renderer = new ReactShallowRenderer();

    renderer.render( <NotFoundPage />);
    expect(renderer).toMatchSnapshot();
});