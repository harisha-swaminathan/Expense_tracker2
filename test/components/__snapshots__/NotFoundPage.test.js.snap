// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component is rendered as expected 1`] = `
ReactShallowRenderer {
  "_context": {},
  "_didScheduleRenderPhaseUpdate": false,
  "_dispatcher": {
    "readContext": [Function],
    "useCallback": [Function],
    "useContext": [Function],
    "useDebugValue": [Function],
    "useDeferredValue": [Function],
    "useEffect": [Function],
    "useId": [Function],
    "useImperativeHandle": [Function],
    "useInsertionEffect": [Function],
    "useLayoutEffect": [Function],
    "useMemo": [Function],
    "useReducer": [Function],
    "useRef": [Function],
    "useResponder": [Function],
    "useState": [Function],
    "useSyncExternalStore": [Function],
    "useTransition": [Function],
  },
  "_element": <NotFoundPage />,
  "_firstWorkInProgressHook": null,
  "_forcedUpdate": false,
  "_idCounter": 0,
  "_instance": null,
  "_isReRender": false,
  "_newState": null,
  "_numberOfReRenders": 0,
  "_renderPhaseUpdates": null,
  "_rendered": <div>
    404! Page not found. 
    <Link
      to="/"
    >
       Go home!
    </Link>
  </div>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": [],
    "_renderer": [Circular],
  },
  "_workInProgressHook": null,
}
`;
